lib/command/info.rb

```diff
@@ -75,21 +63,19 @@ def fetch_app_workloads(org) # rubocop:disable Metrics/MethodLength
       end
 
       if config.app
-        result.select { |app, _| app_matches?(app, config.app, config.current) }
+        result.select { |app, _| config.app_matches?(app, config.app, config.current) }
       else
-        result.reject { |app, _| find_app_options(app).nil? }
+        result.reject { |app, _| config.find_app_config(app).nil? }
       end
     end
 
-    def orgs # rubocop:disable Metrics/MethodLength
+    def orgs
       result = []
 
       if config.org
         result.push(config.org)
       else
-        config.apps.each do |app_name, app_options|
-          next if config.app && !app_matches?(config.app, app_name, app_options)
```

Comment:
This line would never be executed, because providing an `app` without an `org` is not possible (an `app` must have an `org`).